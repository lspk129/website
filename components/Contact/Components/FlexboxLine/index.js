import styled from 'styled-components';

const FlexboxLine = styled.div`
  display: flex;
  justify-content: ${props => (props.between ? 'space-between' : '')};
  flex-wrap: wrap;
  align-items: center;
`;

export default FlexboxLine;
