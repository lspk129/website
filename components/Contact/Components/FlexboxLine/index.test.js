import React from 'react';
import { shallow } from 'enzyme';
import renderer from 'react-test-renderer';

import FlexboxLine from './';

describe('FlexboxLine', () => {
  const shallowWrapper = shallow(<FlexboxLine />);

  it('renders correctly', () => {
    const wrapper = renderer.create(<FlexboxLine />);
    const tree = wrapper.toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('renders div elment', () => {
    expect(shallowWrapper).toBePresent();
    expect(shallowWrapper).toHaveTagName('div');
  });
});
