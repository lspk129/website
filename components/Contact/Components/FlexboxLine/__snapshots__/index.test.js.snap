// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FlexboxLine renders correctly 1`] = `
<div
  className="sc-bdVaJa jVzCaf"
/>
`;
